gremlin-python/src/main/python/example/example.py

```diff
@@ -0,0 +1,158 @@
+        graphson_writer=None, headers=None, session=None,
+        enable_user_agent_on_connect=True
+    )
+    g = traversal().with_remote(rc)
+
+    # cleanup
+    rc.close()
+
+
+def basic_gremlin_example():
+    g = traversal().with_remote(DriverRemoteConnection('ws://localhost:8182/gremlin', 'g'))
+
+    # basic Gremlin: adding and retrieving data
+    v1 = g.add_v('person').property('name', 'marko').next()
+    v2 = g.add_v('person').property('name', 'stephen').next()
+    v3 = g.add_v('person').property('name', 'vadas').next()
+
+    # be sure to use a terminating step like next() or iterate() so that the traversal "executes"
+    # iterate() does not return any data and is used to just generate side-effects (i.e. write data to the database)
+    g.V(v1).add_e('knows').to(v2).property('weight', 0.75).iterate()
+    g.V(v1).add_e('knows').to(v3).property('weight', 0.75).iterate()
+
+    # retrieve the data from the "marko" vertex
+    marko = g.V().has('person', 'name', 'marko').values('name').next()
+    print("name: " + marko)
+
+    # find the "marko" vertex and then traverse to the people he "knows" and return their data
+    people_marko_knows = g.V().has('person', 'name', 'marko').out('knows').values('name').toList()
+    for person in people_marko_knows:
+        print("marko knows " + person)
+
+
+def modern_traversal_example():
+    # This example requires the Modern toy graph to be preloaded upon launching the Gremlin server.
+    # For details, see https://tinkerpop.apache.org/docs/current/reference/#gremlin-server-docker-image and use
+    # conf/gremlin-server-modern.yaml.
+    rc = DriverRemoteConnection('ws://localhost:8182/gremlin', 'g')
+    g = traversal().with_remote(rc)
+
+    e1 = g.V(1).both_e().to_list()  # (1)
+    e2 = g.V(1).both_e().where(__.other_v().has_id(2)).to_list()  # (2)
+    v1 = g.V(1).next()
+    v2 = g.V(2).next()
+    e3 = g.V(v1).both_e().where(__.other_v().is_(v2)).to_list()  # (3)
+    e4 = g.V(v1).out_e().where(__.in_v().is_(v2)).to_list()  # (4)
+    e5 = g.V(1).out_e().where(__.in_v().has(T.id, P.within(2, 3))).to_list()  # (5)
+    e6 = g.V(1).out().where(__.in_().has_id(6)).to_list()  # (6)
+
+    print("1: " + str(e1))
+    print("2: " + str(e2))
+    print("3: " + str(e3))
+    print("4: " + str(e4))
+    print("5: " + str(e5))
+    print("6: " + str(e6))
+
+    # 1. There are three edges from the vertex with the identifier of "1".
+    # 2. Filter those three edges using the where()-step using the identifier of the vertex returned by otherV() to
+    #    ensure it matches on the vertex of concern, which is the one with an identifier of "2".
+    # 3. Note that the same traversal will work if there are actual Vertex instances rather than just vertex
+    #    identifiers.
+    # 4. The vertex with identifier "1" has all outgoing edges, so it would also be acceptable to use the directional
+    #    steps of outE() and inV() since the schema allows it.
```

Comment:
Preferable to use the python-style step syntax
```suggestion
    # 2. Filter those three edges using the where()-step using the identifier of the vertex returned by other_v() to
    #    ensure it matches on the vertex of concern, which is the one with an identifier of "2".
    # 3. Note that the same traversal will work if there are actual Vertex instances rather than just vertex
    #    identifiers.
    # 4. The vertex with identifier "1" has all outgoing edges, so it would also be acceptable to use the directional
    #    steps of out_e() and in_v() since the schema allows it.
```